changelog/change_more_specific_warning_for_style_copyright.md

```diff
@@ -0,0 +1 @@
+* [#12928](https://github.com/rubocop/rubocop/pull/12928): Display a more specific warning for `Style/Copyright` when `AutocorrectNotice` is not set. ([@earlopain][])
```

Comment:
Change log entry is not required just for message coordination. Can you remove it?